### PR TITLE
Create networking fundamentals

### PR DESCRIPTION
**Pull request by stephamon**
Final Alan Hicks whitepaper. 

Takes over #27

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1aabd58aaa/whitepapers/)

File: content/cloud-overview/networking-fundamentals.md [hunk 1]
```diff
@@ -0,0 +1,1162 @@
...
+
+#### Key terms
+
+Before we dig into the details, it's important that we review and define a few terms that you may not have previously learned.
+
+- **node (n.)** - Any single device on a network. A node can be a computer, a network-enabled printer, a router, a managed switch, or something else.
+
+- **bit bucket (n.)** - That place where discarded bits are thrown.
+
+- **copper (n.)** - For purposes of this document, we are only talking about Cat-5 and similar networking cables that send a signal down pairs of copper wires. Many other methods of transmitting data are possible, including fiber-optics, radio waves, even pigeons (for very low bandwidth purposes only of course). Cat-5 and it's derivatives are the most common physical media for transmitting data however, so canonically, copper means Cat-5 and similar cables.
```
Comment (nimbinatus) [inline content/cloud-overview/networking-fundamentals.md]: Sorry, minor nit that my brain can't pass: In the last sentence, `it's` should be `its`

File: content/cloud-overview/networking-fundamentals.md [hunk 2]
```diff
@@ -0,0 +1,1162 @@
...
+
+    201 = (2 * 100) + (0 * 10) + (1 * 1)
+
+Now that you know binary, not only can you count to 31 on one hand, but you can also understand concepts like IP addressing and subnetting.
+
+### Five layers at a glance
+
+The TCP/IP suite makes use of five different layers to get its job done. There *are* a couple of other layers that come into play, but you will rarely run into them unless you are doing exotic things like multi-casting. You can think of the layers in much the same way that you think of a stack of blocks. At the bottom is the physical layer, and at the top is the application layer. Things start at the top and slowly work their way down the layers to create a network frame. Each of these layers will be briefly explained now; we will go into more depth in later sections.
+
+**Physical layer**
```
Comment (nimbinatus) [inline content/cloud-overview/networking-fundamentals.md]: Were these intended to be heading 5s like you have later in the paper? Not sure if these are considered [definitions](http://rackerlabs.github.io/docs-rackspace/style-guide/a-l-style-guidelines.html#glossary-definitions) or headings.

File: content/cloud-overview/networking-fundamentals.md [hunk 3]
```diff
@@ -0,0 +1,1162 @@
...
+
+The lowermost layer in our stack of blocks is the physical layer. This layer consists of basically any physical part of your network. "Physical" is a bit of a misnomer though, as it includes non-physical transmission media such as light or radio signals. Basically anything that is capable of actually transmitting data is part of the physical layer. This includes network cards, copper wires, fiber-optic cable, radio waves, and even infra-red light. The physical layer turns the digital packet into some form of analog signal that can be transmitted to another node on the network.
```
Comment (nimbinatus) [inline content/cloud-overview/networking-fundamentals.md]: `infra-red` is `infrared` ([Source](https://www.merriam-webster.com/dictionary/infrared))

File: content/cloud-overview/networking-fundamentals.md [hunk 4]
```diff
@@ -0,0 +1,1162 @@
...
+
+Today, you'll still hear this terminology from time to time. People often refer to any /24 subnet as a *Class C* network for example.
+
+#### Route determination
+
+Subnetting tells us what IP addresses we should be able to communicate with without going through a router, but what about other IP addresses? Route determination enables us to send information in the form of packets to places that we've never seen.
+
+Every computer has a routing table, though it looks different depending on your operating system. Here's what my routing table currently looks like on Slackware Linux 14.2. Other operating systems format their routing tables differently, but the functionality is the same.
+
+    alan9228@whippoorwill:~# ip route show
```
Comment (nimbinatus) [inline content/cloud-overview/networking-fundamentals.md]: We might want to strip out his username here.

File: content/cloud-overview/networking-fundamentals.md (hunk 2 above)
Comment (stephamon) [inline content/cloud-overview/networking-fundamentals.md]: They're more like definitions. I'll put them in a bulleted list.

File: content/cloud-overview/networking-fundamentals.md (hunk 4 above)
Comment (stephamon) [inline content/cloud-overview/networking-fundamentals.md]: Good catch!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-27a9712418/whitepapers/)

**Pull request merged**